packages/logs/src/domain/logger.ts

```diff
@@ -111,6 +157,18 @@ export class Logger {
     this.log(message, messageContext, StatusType.error, error)
   }
 
+  critical(message: string, messageContext?: object, error?: Error) {
+    this.log(message, messageContext, StatusType.critical, error)
+  }
+
+  alert(message: string, messageContext?: object, error?: Error) {
+    this.log(message, messageContext, StatusType.alert, error)
+  }
+
+  emerg(message: string, messageContext?: object, error?: Error) {
+    this.log(message, messageContext, StatusType.emerg, error)
+  }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:**  since we have a generic log method and that those severities are not commonly used in the browser, what about not exposing additional methods for now?

Reply:
Yes, we agreed on it. It's done.

Reply:
With the team we decided to go for log method and not doing mapping